.changeset/orange-bulldogs-glow.md

```diff
@@ -0,0 +1,15 @@
+---
+"@salt-ds/core": patch
+---
+
+Fixed components text properties impacted by global styles
```

Comment:
did we want to say 'external global styles'? we also have our own global text styles in global.css (applied to the .salt-theme class) which should technically be correct for the below components so it might seem like thats not the case

Reply:
Updated with more details